tests/runtime/other

```diff
@@ -58,6 +58,16 @@ RUN bpftrace -v -e 'BEGIN { if (str($1) != str($2)) { printf("I got %s\n", str($
 EXPECT not equal
 TIMEOUT 5
 
+NAME struct partial string compare - pass
+RUN bpftrace -v -e 'BEGIN { if (strncmp(str($1), str($2), 4)) { printf("I got %s\n", str($1));} exit();}' "hhvm" "hhvm-proc"
+EXPECT I got hhvm
+TIMEOUT 5
+
+NAME struct partial string compare - pass reverse
```

Comment:
Having a more realistic test would be good imo. Something like

`bpftrace -e 't:syscalls:sys_enter_openat /comm == "cat"/ { @[comm]++ }' -c "cat xxx"`